src/accountsdb/db.zig

```diff
@@ -652,11 +665,16 @@ pub const AccountsDB = struct {
             };
             errdefer accounts_file.deinit();
 
+            const file_id = file_info.id;
```

Comment:
this is suppose to be after the `if (n_accounts_this_slot == 0) {` check so we dont add empty files to the map

Reply:
7ff1dd4a3535a6bec16e9bba5e949335f7df88f1